src/interceptors/XMLHttpRequest/XMLHttpRequestOverride.ts

```diff
@@ -515,6 +516,14 @@ export const createXMLHttpRequestOverride = (
       }
     }
 
+    getResponseXML() {
+      const contentType = this.getResponseHeader('Content-Type')
+      if (contentType === 'application/xml' || contentType === 'text/xml') {
```

Comment:
Do you think it's worth to have a more generic content type check like `contentType.endsWith('xml')`, or would it be too insecure on the library's side?

Reply:
There's not big conceptual reason, but I generally prefer checking multiple value equality as `Array.prototype.includes`:

```js
if (['text/xml', 'application/xml'].includes(contentType)) {}
```

Reply:
Hi @kettanaito! Good point. I chose this particular solution because `parseFromString` only accepts a `DOMParserSupportedType` for the second argument. By checking directly against strings, I'm taking advantage of type inference in order to satisfy the TypeScript compiler.

We could still use `.endsWith` or `.includes` and do an explicit type cast like so:
```typescript
const contentType = this.getResponseHeader('Content-Type') ?? ''
if (['application/xml', 'text/xml'].includes(contentType)) {
  return new DOMParser().parseFromString(
    this.responseText,
    contentType as DOMParserSupportedType
  )
}
```

But I thought this was a little less clean. Let me know which approach you think is better, or if you have any other ideas!

Reply:
> Do you think it's worth to have a more generic content type check like `contentType.endsWith('xml')`, or would it be too insecure on the library's side?

I'm not sure -- I don't think I have enough experience to know if this would pose a security issue.